cfgov/jinja2/v1/_includes/molecules/pagination.html

```diff
@@ -47,7 +47,7 @@
             <span class="btn_icon__left
                          cf-icon
                          cf-icon-left"></span>
-            Newer
+            {{ prev }}
```

Comment:
Updated in https://github.com/cfpb/cfgov-refresh/pull/2955/commits/46d56756d3a15495455bddde7eee8ba98b0eeae1